admin-dev/themes/new-theme/js/stock-page/components/products/alert.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <tr>
+    <td colspan="7">
+      <div class="alert alert-info" role="alert">
+        <i class="material-icons">info_outline</i>
+        <p class="alert-text">No product matches your search. Try changing search terms.</p>
+      </div>
+    </td>
+  </tr>
+</template>
+
+<style lang="sass?outputStyle=expanded" scoped>
+  @import "~PrestaKit/scss/custom/_variables.scss";
```

Comment:
if you use "scoped", import the variables on the Main app/component: you are duplicating the file in a unique generated id.